### DISCUSSION TITLE
Recover tmp.mp4.tmp video

### DISCUSSION THREAD

**Discussion by Giuserver** [roleoroleo/yi-hack-Allwinner-v2#369] [Q&A] [answered]
Hello!
It's possible recover a tmp.mp4.tmp video?

Thanks
Giuseppe

**Comment by roleoroleo** [comment, marked as answer]
There is an issue about this.
But I can't find it.

**Reply by Giuserver** [reply]
> There is an issue about this.
> But I can't find it.

i can't find too, also in others hack (v3, v4, allstar ecc..). do you know how can i recover this file?

Thanks

**Reply by roleoroleo** [reply]
What do you mean exactly with "recover"?

**Reply by Giuserver** [reply]
> What do you mean exactly with "recover"?

My first camera is broken. I've recover all files from my sd camera. So, the last minute that camera recorded it's
/tmp/sd/record/2021Y12M20D01H/07M00S60.mp4

Something happened after that minute. but i can't open tmp.mp4.tmp file so i don't know whats happened.

**Reply by roleoroleo** [reply]
Try here:
https://github.com/roleoroleo/yi-hack-Allwinner/issues/52

**Reply by Giuserver** [reply]
> Try here:
> [roleoroleo/yi-hack-Allwinner#52](https://github.com/roleoroleo/yi-hack-Allwinner/issues/52)

Thankyou!
https://www.stellarinfo.com/disk-recovery/video-repair.php
This work in part! (this program close the file tmp.mp4.tmp he need just an example file)
Anyway, i think it's more easy! the file it's not corrupted! it's not complete .. but i don't know which part of the hack can i use for complete it. Any good ideas it will be great
